bower.json

```diff
@@ -30,7 +30,7 @@
     "jsonlint": "~1.6.0",
     "ng-clip": "~0.2.6",
     "ng-lodash": "~0.2.3",
-    "ng-password-strength": "~0.1.0",
+    "ng-password-strength": "~0.2.1",
```

Comment:
Becouse in  version  > 0.2.0 dependency on bootstrap was removed. And it is fine.